src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -371,8 +371,14 @@ private bool TryFindInGAC(AssemblyName assemblyName, out string assemblyFilePath
                 {
                     if (string.IsNullOrEmpty(_gacPath64))
                     {
-                        // cache value of '_gacPath64' folder in member variable.
-                        _gacPath64 = $"{_winDir}{dirSeparator}Microsoft.NET{dirSeparator}assembly{dirSeparator}GAC_64";
+                        if (RuntimeInformation.ProcessArchitecture == Architecture.Arm64)
+                        {
+                            _gacPath64 = $"{_winDir}{dirSeparator}Microsoft.NET{dirSeparator}assembly{dirSeparator}GAC_Arm64";
+                        }
+                        else
+                        {
+                            _gacPath64 = $"{_winDir}{dirSeparator}Microsoft.NET{dirSeparator}assembly{dirSeparator}GAC_64";
+                        }
```

Comment:
Interpolated string uses implicitly CurrentCulture but our convention is (1) use InvariantCulture in Engine, (2) use culture _explicitly_.

```suggestion
                       var gacName = RuntimeInformation.ProcessArchitecture == Architecture.Arm64 ? "GAC_Arm64": "GAC_64";
                         _gacPath64 = string.Create(CultureInfo.InvariantCulture, $"{_winDir}{dirSeparator}Microsoft.NET{dirSeparator}assembly{dirSeparator}{gacName}";
```